tests/spec/test_enum.py

```diff
@@ -170,3 +169,37 @@ def test_has_thrift_module(loads):
         }
     ''')
     assert module is module.Foo.__thrift_module__
+
+
+def test_primitive_type_string(loads):
+    Answer = loads('''
+        enum Answer {
+            Yes, No, Unspecified
+        } (py.primitiveType = "string")
+    ''').Answer
+
+    spec = Answer.type_spec
+
+    assert spec.to_primitive(Answer.Yes) == 'Yes'
+    assert spec.to_primitive(Answer.No) == 'No'
+    assert spec.to_primitive(Answer.Unspecified) == 'Unspecified'
```

Comment:
Node doesn't need to do this because Node's runtime representation of enums is already the string version (CC @kriskowal). For Go, enums are already a separate type (with their own `String()`) method so again, they don't need to do this.